yt_dlp/extractor/kenh14.py

```diff
@@ -0,0 +1,150 @@
+    _TESTS = [{
+        'url': 'https://video.kenh14.vn/video/mo-hop-iphone-14-pro-max-nguon-unbox-therapy-316173.chn',
+        'md5': '1ed67f9c3a1e74acf15db69590cf6210',
+        'info_dict': {
+            'id': '316173',
+            'ext': 'mp4',
+            'title': 'Video mở hộp iPhone 14 Pro Max (Nguồn: Unbox Therapy)',
+            'description': 'Video mở hộp iPhone 14 Pro MaxVideo mở hộp iPhone 14 Pro Max (Nguồn: Unbox Therapy)',
+            'thumbnail': r're:^https?://videothumbs\.mediacdn\.vn/.*\.jpg$',
+            'tags': [],
+            'uploader': 'Unbox Therapy',
+            'upload_date': '20220517',
+            'view_count': int,
+            'duration': 722.86,
+            'timestamp': 1652764468,
+        },
+    }, {
+        'url': 'https://video.kenh14.vn/video-316174.chn',
+        'md5': '2b41877d2afaf4a3f487ceda8e5c7cbd',
+        'info_dict': {
+            'id': '316174',
+            'ext': 'mp4',
+            'title': 'Khoảnh khắc VĐV nằm gục khóc sau chiến thắng: 7 năm trời Việt Nam mới có HCV kiếm chém nữ, chỉ có 8 tháng để khổ luyện trước khi lên sàn đấu',
+            'description': 'md5:de86aa22e143e2b277bce8ec9c6f17dc',
+            'thumbnail': r're:^https?://videothumbs\.mediacdn\.vn/.*\.jpg$',
+            'tags': [],
+            'upload_date': '20220517',
+            'view_count': int,
+            'duration': 70.04,
+            'timestamp': 1652766021,
+        },
+    }, {
+        'url': 'https://video.kenh14.vn/0-344740.chn',
+        'md5': 'b843495d5e728142c8870c09b46df2a9',
+        'info_dict': {
+            'id': '344740',
+            'ext': 'mov',
+            'title': 'Kỳ Duyên đầy căng thẳng trong buổi ra quân đi Miss Universe, nghi thức tuyên thuệ lần đầu xuất hiện gây nhiều tranh cãi',
+            'description': 'md5:2a2dbb4a7397169fb21ee68f09160497',
+            'thumbnail': r're:^https?://kenh14cdn\.com/.*\.jpg$',
+            'tags': ['kỳ duyên', 'Kỳ Duyên tuyên thuệ', 'miss universe'],
+            'uploader': 'Quang Vũ',
+            'upload_date': '20241024',
+            'view_count': int,
+            'duration': 198.88,
+            'timestamp': 1729741590,
+        },
+        'expected_warnings': [
+            'Failed to download m3u8 information: HTTP Error 404: NOT FOUND',
+        ],
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
+        attrs = extract_attributes(get_element_html_by_attribute('type', 'VideoStream', webpage) or '')
+        direct_url = attrs['data-vid']
+
+        metadata = self._download_json(
+            'https://api.kinghub.vn/video/api/v1/detailVideoByGet?FileName={}'.format(
+                remove_start(direct_url, 'kenh14cdn.com/')), video_id, fatal=False)
+
+        return {
+            'id': video_id,
+            'title': (
+                strip_or_none(metadata.get('title'))
```

Comment:
if the `metadata` request is non-fatal then we need to use `traverse_obj` here
```suggestion
                traverse_obj(metadata, ('title', {strip_or_none}))
```